egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -18,8 +18,8 @@
 import shutil
 import math
 
-train_lib = imp.load_source('ntl', 'steps/nnet3/nnet3_train_lib.py')
-chain_lib = imp.load_source('ncl', 'steps/nnet3/chain/nnet3_chain_lib.py')
+common_train_lib = imp.load_source('ntl', 'steps/nnet3/libs/common_train_lib.py')
```

Comment:
@vimalmanohar since the introduction of the xconfig scripts the python code is getting more complex, as we will now be maintaining "layers" as python classes and not as simple string generating methods. In  https://github.com/danpovey/kaldi/pull/30 we have started packaging the python scripts. See if you can follow the same thing here to package the training libraries. 

Reg. imports : As we start packaging the python modules we could also start using the normal import statements rather than using `imp.load_source`, which is usually not well known. See https://github.com/danpovey/kaldi/pull/30 on how to modify the `sys.path` to add the local python packages to search path.